lib/osf-components/addon/components/resources-list/edit-resource/component.ts

```diff
@@ -38,11 +38,19 @@ export default class EditResourceModal extends Component<Args> {
             presence: true,
             translationArgs: { description: this.intl.t('osf-components.resources-list.edit_resource.doi') },
         })],
-        resourceType: [validatePresence({
-            type: 'blank',
-            presence: true,
-            translationArgs: { description: this.intl.t('osf-components.resources-list.edit_resource.output_type') },
-        })],
+        resourceType: [
+            validatePresence({
+                type: 'blank',
+                presence: true,
+                translationArgs: {
+                    description: this.intl.t('osf-components.resources-list.edit_resource.output_type'),
+                },
+            }),
+            validateExclusion({
+                list: ['undefined'],
+                type: 'mustSelect',
+            }),
```

Comment:
Also, where is the error message coming from? Is it directly from the addon, or are we providing a translation string for it somewhere down the line?

Reply:
https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/translations/en-us.yml#L478